api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryService.java

```diff
@@ -26,5 +33,61 @@
 
   List<ReportingUser> getUsers();
 
-  List<ReportingWorkspace> getWorkspaces();
+  List<ReportingWorkspace> getWorkspaces(long limit, long offset);
+
+  default List<ReportingWorkspace> getWorkspaces() {
+    return getAll(this::getWorkspaces);
+  }
+
+  default Stream<List<ReportingWorkspace>> getWorkspacesStream() {
```

Comment:
I'll probably make one of these for every table, which makes things very nice on the consumption end.